src/app/components/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { RollerSpinner } from './spinner.styles';
+
+const Spinner = () => {
```

Comment:
instead of `() => {return(<Spinner></Spinner>)}`
you can do shorter version
`() => (<Spinner></Spinner>)`